docker-compose.yml

```diff
@@ -78,3 +78,9 @@ services:
       - "traefik.docker.network=frontend"
       - "traefik.http.routers.${COMPOSE_PROJECT_NAME}mail.rule=Host(`mail-${COMPOSE_DOMAIN}`)"
       - "traefik.http.services.${COMPOSE_PROJECT_NAME}mail.loadbalancer.server.port=8025"
+
+  node:
+    image: node:18
+    volumes:
+      - .:/app
+    working_dir: /app
```

Comment:
We already have this in [`docker-compose.override.yml`](https://github.com/itk-dev/dokk1gh/blob/develop/docker-compose.override.yml):

```suggestion
```